core/src/main/java/dk/alexandra/fresco/logging/DefaultPerformancePrinter.java

```diff
@@ -0,0 +1,16 @@
+package dk.alexandra.fresco.logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultPerformancePrinter implements PerformancePrinter {
+
+  public Logger log = LoggerFactory.getLogger(PerformanceLogger.class);
```

Comment:
Probably this should be `public Logger log = LoggerFactory.getLogger(DefaultPerformancePrinter.class);`. Not `PerformanceLogger.class`.